tests/lib/rules/no-unused-vars.js

```diff
@@ -1132,6 +1132,11 @@ ruleTester.run("no-unused-vars", rule, {
             options: [{ caughtErrors: "all", caughtErrorsIgnorePattern: "^ignore" }],
             errors: [definedError("err", ". Allowed unused args must match /^ignore/u")]
         },
+        {
+            code: "try{}catch(err){};",
+            options: [{ caughtErrors: "all", varsIgnorePattern: "^err" }],
+            errors: [definedError("err", ". Allowed unused vars must match /^err/u")]
```

Comment:
Since `varsIgnorePattern` should not apply to the catch block, the error message should not contain `Allowed unused vars must match /^err/u"`

Reply:
Done!